mne/io/nirx/nirx.py

```diff
@@ -16,6 +16,7 @@
 from ...transforms import apply_trans, _get_trans
 from ...utils import logger, verbose, fill_doc
 from ...utils import warn
+from ...preprocessing import annotate_nan
```

Comment:
This line is causing problems during imports. There is an import loop while importing `preprocessing` modules :

    File "c:\users\juliend\documents\mne\mne\io\nirx\__init__.py", line 7, in <module>
        from .nirx import read_raw_nirx
    File "c:\users\juliend\documents\mne\mne\io\nirx\nirx.py", line 19, in <module>
        from ...preprocessing import annotate_nan
    File "c:\users\juliend\documents\mne\mne\preprocessing\__init__.py", line 11, in <module>
        from .maxfilter import apply_maxfilter
    File "c:\users\juliend\documents\mne\mne\preprocessing\maxfilter.py", line 10, in <module>
        from ..io import read_raw_fif
    ImportError: cannot import name 'read_raw_fif' from partially initialized module 'mne.io' (most likely due to a circular    import) (c:\users\juliend\documents\mne\mne\io\__init__.py)

Don't know how to resolve it since it doesnt seem to be linked to me...?

Reply:
Could you put it with the other trouble maker here? 

https://github.com/mne-tools/mne-python/blob/806cd4785f927dbf262b7c878a2a8f143ee64a63/mne/io/nirx/nirx.py#L106

Reply:
It fixed the problem, thanks 🙂

Reply:
Glad it worked 👍